packages/cloudflare/src/cli/build/build-next-app.ts

```diff
@@ -7,17 +8,15 @@ import { execSync } from "node:child_process";
  *
  * @param nextAppDir the directory of the app to build
  */
-export function buildNextjsApp(nextAppDir: string): void {
-  runNextBuildCommand("pnpm", nextAppDir);
+export async function buildNextjsApp(nextAppDir: string): Promise<void> {
+  const pm = await detect();
+
+  runNextBuildCommand(pm?.name ?? "npm", nextAppDir);
```

Comment:
Should the code throw if not pm is detected?

Reply:
sure 👍

Reply:
updated 🙂